pyartcd/pyartcd/pipelines/update_golang.py

```
@@ -972,6 +979,36 @@ def _get_doozer_group_and_image(self, el_v, go_version):
+            branch = self.GOLANG_DATA_BRANCH
+            image_key = self.get_golang_image_key(el_v, go_version)
+        else:
+            branch = self.get_golang_branch(el_v, go_version)
+            image_key = GOLANG_BUILDER_IMAGE_NAME
+
+        ref = self.data_gitref if self.data_gitref else branch
+        repo = self._get_upstream_ocp_build_data_repo()
+
+        try:
+            image_config = self._load_yaml_from_repo(repo, f"images/{image_key}.yml", ref)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)